crates/sui-analytics-indexer/src/store/bq/schemas/wrapped_object.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE IF NOT EXISTS chaindata.WRAPPED_OBJECT
+(
+    object_id              STRING,
+    root_object_id         STRING        NOT NULL,
+    root_object_version    INT64         NOT NULL,
+    checkpoint             INT64         NOT NULL,
+    epoch                  INT64         NOT NULL,
+    timestamp_ms           INT64         NOT NULL,
+    json_path              STRING,
+    struct_tag             STRING
+)
+    PARTITION BY RANGE_BUCKET(epoch, GENERATE_ARRAY(0, 100000, 10))
```

Comment:
how is partitioning used here? iirc if an object gets wrapped, unwrapped and wrapped again at a different epoch, the `epoch` column gets updated and as a result the row will be moved to a different partition, this seems a bit wasteful.

Reply:
Sorry I do not follow the comment about what is wasteful. Here is more info on partitioned tables https://cloud.google.com/bigquery/docs/partitioned-tables. All our tables are partitionted by epoch but clustered differently (depends on the table)